src/Org.Quickstart.API/Controllers/HealthCheckController.cs

```diff
@@ -1,21 +0,0 @@
-﻿using System;
```

Comment:
why would you remove the healthcheck controller.  It's a sanity check and a heartbeat when you cluster.